src/dgea/deseq2/script.py

```diff
@@ -0,0 +1,186 @@
+import scanpy as sc
+import mudata as mu
+import pandas as pd
+import numpy as np
+import scipy.sparse as sp
+from pydeseq2.dds import DeseqDataSet
+from pydeseq2.ds import DeseqStats
```

Comment:
I very strongly suggest you use the original R DEseq2 package instead of this Python version (for various reasons)